<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aula 18 - Combo Box</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{background-color: rgb(255, 255, 255);border:0;font-weight:bold;font-size:12px;padding: 3px 10px;}</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>16</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Campos dos Goytacazes - RJ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ribeir√£o Preto - SP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New York - NY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Los Angeles - CA</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>229</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Escolha uma cidade:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>292</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:18px;font-weight:bold; color:rgb(66, 199, 255);</string>
    </property>
    <property name="text">
     <string>Cidade selecionada</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border:0; background-color:#FFF;border-radius:20px;font-weight:bold;font-size:12px;}QPushButton:hover{background-color:rgb(211, 211, 211);}</string>
    </property>
    <property name="text">
     <string>Selecionar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
